x/ibc/mock/bank/client/cli/tx.go

```diff
@@ -0,0 +1,128 @@
+package cli
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
+	ics23 "github.com/cosmos/cosmos-sdk/x/ibc/23-commitment"
+	"github.com/cosmos/cosmos-sdk/x/ibc/mock/bank/internal/types"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func GetTxCmd(cdc *codec.Codec) *cobra.Command {
+	txCmd := &cobra.Command{
+		Use:   "ibcmockbank",
+		Short: "IBC mockbank module transaction subcommands",
+		// RunE:  client.ValidateCmd,
+	}
+	txCmd.AddCommand(
+		GetTransferTxCmd(cdc),
+		GetMsgRecvPacketCmd(cdc),
+	)
+
+	return txCmd
+}
+
+func GetTransferTxCmd(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "transfer --src-port <src port> --src-channel <src channel> --denom <denomination> --amount <amount> --receiver <receiver> --source <source>",
```

Comment:
Any required flags need to be converted to args. Something like: 

```
transfer [src-port] [src-channel] [denom] [amount] [receiver] [source]
```

Reply:
Also `denom` and `amount` should be one arg (we can parse them out) and `[receiver]` should something like `[to-addr]`. Also what is the purpose of source?

Reply:
> Also `denom` and `amount` should be one arg (we can parse them out) and `[receiver]` should something like `[to-addr]`. Also what is the purpose of source?

The source is used to mark whether the sending chain is the source of the asset in ICS 20. If source is true, will escrow tokens on the sending chain.

Reply:
Maybe a better way to do that is on the chain instead of specified in msg.